src/hooks/useTable.js

```diff
@@ -396,6 +396,12 @@ export const useTable = (props, ...plugins) => {
 
   getInstance().prepareRow = React.useCallback(
     row => {
+      if (row.preparedWith === getInstance().state) {
+        return
+      }
+
+      row.preparedWith = getInstance().state
```

Comment:
Helpful when virtualization is done, this helps in skipping prepareRow call if it has already been called once with same table state..